README.md

```diff
@@ -66,23 +65,24 @@ $ openssl req -new -newkey rsa:4096 -x509 -sha256 -days 365 -nodes -out server.c
 
 this will create 2 files: `server.crt` and `server.key`
 
-2. start hlx with tls support
+2. start aem with tls support
 
 ```
-$ hlx up --tls-cert server.crt --tls-key server.key
-    ____              __    ___   v14.21.4
-   / __/______ ____  / /__ / (_)__
-  / _// __/ _ `/ _ \/  '_// / / _ \
- /_/ /_/  \_,_/_//_/_/\_\/_/_/_//_/
-
-info: Starting Franklin dev server v14.21.4
-info: Local Franklin dev server up and running: https://localhost:3000/
+$ aem up --tls-cert server.crt --tls-key server.key
+    ___   ______  ___  ____   __         ___      ___
+   / _ | / __/  |/  / / __/__/ /__ ____ / _ \___ / (_)  _____ ______ __
+  / __ |/ _// /|_/ / / _// _  / _ `/ -_) // / -_) / / |/ / -_) __/ // /
+ /_/ |_/___/_/  /_/ /___/\_,_/\_, /\__/____/\__/_/_/|___/\__/_/  \_, /
+                             /___/                      v14.26.0 ___/
```

Comment:
Can you remove the AEM? The official name is "Edge Delivery Services for Adobe Experience Manager"
cc @trieloff

Reply:
Analog to the Importer, maybe here it should just say Simulator?

Reply:
ok.

Reply:
changed it and updated the description.